prop.properties

```diff
@@ -0,0 +1,8 @@
+url = https://chpl.ahrqdev.org/
+apikey = 12909a978483dfb8ecd0596c98ae9094
+roleAdminUsername= aqa-admin
+roleAdminPassword= A good long password123
```

Comment:
Should never commit passwords in git, and also should not push them to github. This is a public repository and once a password is pushed it's public. I've disabled all of the AQA users, and we'll need to change their passwords to different ones before they can be re-enabled